docs/metal_interconnections_create.md

```diff
@@ -24,14 +24,17 @@ metal interconnections create -n <name> [-m <metro>] [-r <redundancy> ] [-t <typ
 ### Options
 
 ```
-  -h, --help                    help for create
-  -m, --metro string            metro in the interconnection
-  -n, --name string             Name of the interconnection
-  -O, --organizationID string   Org ID
-  -p, --projectID string        project ID
-  -r, --redundancy string       Website URL of the organization.
-  -t, --type string             type of of interconnection.
-  -v, --vrfs strings            Return only the specified vrfs.
+  -h, --help                        help for create
+  -m, --metro string                metro in the interconnection
+  -n, --name string                 Name of the interconnection
+  -O, --organizationID string       Org ID
+  -p, --projectID string            project ID
+  -r, --redundancy string           Website URL of the organization.
```

Comment:
This doesn't seem right. (redundacy is a string with a URL as a parameter?)

Reply:
Yes, that seems incorrect. It seems the already existing flag descriptions for most of them needs to be corrected